lua/sg/cody/rpc.lua

```diff
@@ -265,8 +305,7 @@ end
 ---@return table | nil
 M.execute.code_question = function(message, callback)
   local message_id = utils.uuid()
-
-  M.message_callbacks[message_id] = callback
+  chat_message_handlers[message_id] = callback
```

Comment:
Missed one of these renames in `lua/sg/components/layout/hover.lua`

Reply:
Well so I wanted to remove the message_callbacks variable in the module, but I see that we're using it in `hover.lua`. Can you tell me why we're doing that? I'm not sure I understand what's happening there